CONTRIBUTING.md

```diff
@@ -121,7 +121,7 @@ requirements for inclusion in the package:
 
 **Code Quality**
   * Are the [coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html) followed?
-  * Is the code compatible with Python >=3.8?
+  * Is the code compatible with Python >=3.10?
   * Are there dependencies other than the `astropy` core, the Python Standard
     Library, and NumPy 1.18.0 or later?
```

Comment:
This is badly out of date. Perhaps it's best to replace an explicit listing of dependencies with a reference to `pyproject.toml`.

Reply:
✅